oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/requests/factory/v7/V7DocumentFactory.java

```diff
@@ -164,7 +164,7 @@ public HttpRequest update(UpdateRequest request, Map<String, ?> params) {
         }
         final byte[] content = version.codec().encode(ImmutableMap.of("doc", doc));
 
-        builder.post("/{index}/_doc/{id}/_update")
+        builder.post("/{index}/_update/{id}")
```

Comment:
This is a v7 factory, why change this? You mentioned v8.1.

Reply:
> This is a v7 factory, why change this? You mentioned v8.1.

I think the related issue mentioned that both 7.x and 8.x support this, the previous one is deprecated, right @dingzx3 ? I'll start the CI to see

Reply:
Let's confirm from ElasticSearch API docs as well.

Reply:
The last one supported since 7.3，the previous one is  deprecated and removed in 8.1.

Reply:
> The last one supported since 7.3

Which one? I think we need to support the whole 7.x ElasticSearch?

Reply:
> The last one supported since 7.3，the previous one is  deprecated and removed in 8.1.

Can you please add a new codec for 8.x and keep the original one as is?